security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -146,7 +148,7 @@ type Debug struct {
 }
 
 // newSDSService creates Secret Discovery Service which implements envoy v2 SDS API.
-func newSDSService(st cache.SecretManager, skipTokenVerification, localJWT bool,
+func newSDSService(st cache.SecretManager, skipTokenVerification, localJWT, fileMountedCerts bool,
```

Comment:
Ingress gateway can support fetching from secret + file mount, right? @JimmyCYJ 
This flag means only file mount is used? If so, we should make it explicit in the flag. Using a name like "fileMountedCertsOnly".

Reply:
I am not sure if such a case exists. But there is no harm in changing the var name. So I changed